cli/tools/lint/rules/no_sloppy_imports.md

```diff
@@ -0,0 +1,20 @@
+Enforces specifying explicit references to paths in module specifiers.
+
+Non-explicit specifiers are ambiguous and require probing for the correct file
+path on every run, which has a performance overhead.
+
+Note: This lint rule is only active when using `--unstable-sloppy-imports`.
+
+### Invalid:
+
+```typescript
+import { add } from "./math/add";
```

Comment:
Cool that this will be showed in doc help output, but it won't be shown on https://lint.deno.land/. Any ideas how we could do that?

Reply:
Actually I think we don't have markdown help text output for specific rules anymore in the `deno` binary...

Reply:
I think we need to move lint.deno.land out of the deno_lint repo and have it automatically update on publish of the CLI. I opened https://github.com/denoland/deno_lint/issues/1300